code/game/objects/items/stacks/medical.dm

```diff
@@ -1,24 +1,34 @@
+#define REAGENT_AMOUNT_PER_ITEM 20 //The amount of reagents medical items contain, for both application and grinding purposes.
+
 /obj/item/stack/medical
 	name = "medical pack"
 	singular_name = "medical pack"
 	icon = 'icons/obj/stacks/miscelaneous.dmi'
-	amount = 6
-	max_amount = 6
+	amount = 12
+	max_amount = 12
 	w_class = WEIGHT_CLASS_TINY
 	full_w_class = WEIGHT_CLASS_TINY
 	throw_speed = 3
 	resistance_flags = FLAMMABLE
 	max_integrity = 40
 	novariants = FALSE
-	///How much brute does it heal?
-	var/heal_brute = 0
-	///How much burn does it heal?
-	var/heal_burn = 0
+	///What reagent does it apply?
+	var/list/reagent
+	///Is this for bruises?
+	var/heal_brute = FALSE
+	///Is this for burns?
+	var/heal_burn = FALSE
 	///For how long does it stop bleeding?
 	var/stop_bleeding = 0
 	///How long does it take to apply on yourself?
 	var/self_delay = 2 SECONDS
 
+/obj/item/stack/medical/Initialize(mapload, new_amount, merge, mob/user)
+	. = ..()
+	if(reagent)
+		create_reagents(REAGENT_AMOUNT_PER_ITEM)
```

Comment:
I think it's fine to be 40u if you accept the change I suggested for how reagent works

Reply:
That would be 40u divided between the whole stack though wouldn't it? That would be an extraordinary nerf over the ability to be treated with a first aid kit.